src/SegmentedMsg.chpl

```diff
@@ -781,7 +781,10 @@ module SegmentedMsg {
                 select gIV.dtype {
                     when DType.Int64 {
```

Comment:
is it worthwhile to update the `DType.UInt64` case too?

Reply:
I don't think that code path can be hit by the data frame indexing message case and I just ran a quick `writeln` test to make sure, so I don't think that would help this specific effort, but could be added if we choose to support this small array approach elsewhere.